### PR TITLE
Issue #3 - Set up redux: done, create and connect store, middlewares,…

### PR DESCRIPTION
**Pull request by myroslav-kolomiiets**
… add support of redux-dev-tools, add root reducer

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,6 +3,24 @@ import React, {Component} from 'react';
 import logo from './logo.svg';
 import './App.css';
 
+const TEST_WORKING = 'TEST_WORKING';
+
+const initState = {
+    isWorking: true
+};
+
+export const test = (state = initState, action) => {
```
Comment (zonzujiro) [inline src/App.js]: Pls, create module in  `modules` for this one

File: src/store/reducers.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import {combineReducers} from 'redux';
```
Comment (zonzujiro) [inline src/store/reducers.js]: I think a better place for this one is in `modules` folder because all our reducers will be there

**Review by zonzujiro — CHANGES_REQUESTED** [review]
Good job, but see comments

**Comment by zonzujiro** [comment]
Looks like you forgot to use styles of prettier. In other - lgtm

File: src/App.js (hunk 1 above)
Comment (myroslav-kolomiiets) [reply src/App.js]: Done

File: src/store/reducers.js (hunk 2 above)
Comment (myroslav-kolomiiets) [reply src/store/reducers.js]: Done

**Review by zonzujiro — APPROVED** [review]
🚢

**Pull request merged**